docs/cli.md

```diff
@@ -480,17 +449,20 @@ bigflow deploy-image \
 
 #### Complete deploy examples
 
-Upload DAG files from `{current_dir}/.dags` dir and a Docker image from a local repository using `local_account` authentication.
+Upload DAG files from `{current_dir}/.dags` dir and a Docker image from `{current_dir}/image` dir using `local_account` authentication.
 Configuration is taken from `{current_dir}/deployment_config.py`:
 
 ```bash
-bigflow deploy --version 1.0 --config dev
+bigflow deploy --config dev
 ```
 
-The same, but configuration is taken from a given file:
+The same, but configuration and docker image are taken from specified files:
```

Comment:
The same, but a configuration and a docker image are taken from the specified files.